config/clusters/opensci/cluster.yaml

```diff
@@ -0,0 +1,23 @@
+name: opensci
+provider: aws
```

Comment:
```suggestion
provider: aws # https://2i2c.awsapps.com/start#/
```

Reply:
Is `account` only needed in cluster.yaml when we don't assume our aws management account is used to manage this cluster's AWS account?

>```python
>        # We mostly use our 2i2c account to login into cloud provider UI's
>        # with a few exceptions
>        account = cluster.get("account", "2i2c")
>```

Reply:
@consideRatio, yes.